helio_tools/_src/utils/time.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+
+def check_datetime_format(datetime_str: str) -> datetime:
+    try:
+        datetime_obj = datetime.strptime(datetime_str, "%Y-%m-%d")
+        return datetime_obj
+    except ValueError:
+        msg = "Datetime string format is incorrect."
+        msg += f"\nInput: {datetime_str}"
+        print(msg)
+
+
+def check_datetime_format_solo(datetime_str: str) -> datetime:
+    try:
+        datetime_obj = datetime.strptime(datetime_str, "%Y-%m-%d %H:%M")
+        return datetime_obj
+    except ValueError:
+        msg = "Datetime string format is incorrect."
+        msg += f"\nInput: {datetime_str}"
+        print(msg)
```

Comment:
Perhaps add a boolean in the original check_datetime_format to avoid duplicate code, 


`check_datetime_format_solo(datetime_str: str, is_solo: boolean = False) -> datetime`

Reply:
Good point. Added a dictionary of predefined formats to avoid duplicate code.